tools/build_image.py

```diff
@@ -27,20 +28,21 @@
     ],
     UserData=USER_DATA
 )
-print('   Instance created. ID: {}'.format(instances[0].id))
+print(f'   Instance created. ID: {instances[0].id}')
 
 
 # Wait for EC2 instance to be 'running'
 
 print('Waiting for AWS EC2 instance state to be "running"')
+time.sleep(5)
```

Comment:
Maybe is there a way to listen to the instance creation? Instead of using `sleep`? Like a JS promise?

Reply:
We already do that in this function ` wait_for_instance_running` but it works only if the package works well

Reply:
What do you mean about it?